.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
All of this is delightful discussion to have on the nvm repo when the time comes. For now, I'm suggesting "node" if we don't want to have to manually bump it to "7" etc in the future. If we don't like that, so be it.